### PR TITLE
RELATED: RAIL-2725 update changelog for SDK8

### PR DESCRIPTION
**Pull request by no23reason**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,19 @@ The REST API versions in the table are just for your information as the values a
...
+
+### Added
+
+- Infrastructure improvements
+  - Version 8.0 comes with significant changes in almost all aspects of the library. While a lot of the changes can be categorized as infrastructural, Version 8.0 also delivers a number of features aimed at developer productivity and ease of use. For the full list of new things, see [What's New in Version 8.0](https://sdk.gooddata.com/gooddata-ui/docs/whats_new_8.html).
+- Ranking filters
+  - You can now use ranking filters to filter your visualizations. A ranking filter shows the data from a set of the highest or lowest ranked values of an attribute based on the ranking criteria that you establish. For more information, see [Ranking Filter](https://sdk.gooddata.com/gooddata-ui/docs/ranking_filter_component.html).
+- Turning off data points
+  - You can now turn off data points in line charts, area charts and combo charts. For more information, see [Chart Config -> Configure canvases](https://sdk.gooddata.com/gooddata-ui/docs/chart_config.html#configure-canvases).
+
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md:32]: - Infrastructure improvements and a number of features aimed at developer productivity and ease of use ([doc](https://sdk.gooddata.com/gooddata-ui/docs/whats_new_8.html))
- Ranking filters that show the data from a set of the highest or lowest ranked values of an attribute based on the ranking criteria that you establish ([doc](https://sdk.gooddata.com/gooddata-ui/docs/ranking_filter_component.html))
- The option to turning off data points in line charts, area charts, and combo charts ([doc](https://sdk.gooddata.com/gooddata-ui/docs/chart_config.html#configure-canvases))

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8209/ : SUCCESS in 5m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4532/ : SUCCESS in 3s

**Review by kandl — APPROVED** [review]
LGTM

**Review by lupko — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#8211](https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8211/))
Covered lines: **91.97%**, change: **0%**
Covered branches: **84.05%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8212/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4314/ : SUCCESS in 13m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/214/ : FAILURE in 3m 03s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8212/ : SUCCESS in 4m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1994/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/3034/ : SUCCESS in 17m 51s

**Pull request merged**